src/link-not-found.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Course link not found
```

Comment:
We should add other config here. See what we do on `404`.

Reply:
Good catch, thanks!